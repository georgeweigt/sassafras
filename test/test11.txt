* From SAS Manual p. 5437
title 'Simple Linear Regression';
data Class;
input Name $ Height Weight Age @@;
datalines;
Alfred 69.0 112.5 14 Alice 56.5 84.0 13 Barbara 65.3 98.0 13
Carol 62.8 102.5 14 Henry 63.5 102.5 14 James 57.3 83.0 12
Jane 59.8 84.5 12 Janet 62.5 112.5 15 Jeffrey 62.5 84.0 13
John 59.0 99.5 12 Joyce 51.3 50.5 11 Judy 64.3 90.0 14
Louise 56.3 77.0 12 Mary 66.5 112.0 15 Philip 72.0 150.0 16
Robert 64.8 128.0 12 Ronald 67.0 133.0 15 Thomas 57.5 85.0 11
William 66.5 112.0 15
;

proc reg;
model Weight = Height;
run;
