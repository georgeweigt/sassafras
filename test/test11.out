                            Simple Linear Regression

                              Analysis of Variance

               DF     Sum of Squares     Mean Square     F Value     Pr > F
     Model      1         7193.24912      7193.24912       57.08     0.0000
     Error     17         2142.48772       126.02869                       
     Total     18         9335.73684                                       

              Root MSE            11.22625     R-Square     0.7705
              Dependent Mean     100.02632     Adj R-Sq     0.7570
              Coeff Var           11.22330                        

                              Parameter Estimates

                         Estimate      Std Err     t Value     Pr > |t|
         Intercept     -143.02692     32.27459       -4.43       0.0004
         HEIGHT           3.89903      0.51609        7.55       0.0000

